cmd/wallet-web/src/pages/OIDCShare.vue

```diff
@@ -318,6 +318,21 @@ export default {
       const encodedSignature = encode(signature).slice(0, -1);
       this.idToken = `${encodedHeader}.${encodedPayload}.${encodedSignature}`;
     },
+    async generateVPToken() {
+      const { controller } = this.getCurrentUser().preference;
+      const { presentation } = await this.credentialManager.present(
+        this.token,
+        { rawCredentials: this.presentations[0].verifiableCredential },
+        { controller }
+      );
+      // id is not given by API
+      presentation.id = presentation.verifiableCredential[0].id;
+      // proof is not needed at this phase
+      presentation.proof = [];
+      // no need to provide verifiable credential in vp token at this phase
+      delete presentation.verifiableCredential;
```

Comment:
Should be fixed

Reply:
Fixed, but need to fix the JSON input issue